### ISSUE TITLE
Section: Introduction

### ISSUE THREAD

**Issue by mmore500** [chaynes2019/AvidaGeneDupe#17]
- [x] revise to match new results

**Comment by amlalejini** [comment]
- [ ] as part of revising to match new results, update literature review
- [ ] depending on target venue, remove / de-emphasize evolutionary computation discussion in intro

**Comment by mmore500** [comment]
> Although comparative studies lack definitive proof of laboratory manipulation, the accumulated weight of evidence clearly suggests that gene and genome duplication events can have long-term evolutionary consequences.

- [ ] `multiplayaccess` Curious if there are laboratory manipulation studies now? (can do some digging when taking a pass on related literature)

**Comment by mmore500** [comment]
> The prominence of gene duplications in biological evolution has inspired their use in artificial evolutionary systems.
> In genetic programming, the optimal program architecture for solving a particular problem is challenging to predict \textit{a priori}.
> Inspired by Ohno's \textit{Evolution by Gene Duplication} \citep{ohno1970evolution}, Koza used gene duplication and deletion operators to co-evolve genetic programs and their structure, finding that gene duplication and deletion increased program evolvability and produced simpler solutions \citep{Koza:1995fr}.
> Calabretta \textit{et al.} evolved modular neural network motor controllers for robots with and without module duplication operators, finding evidence that access to duplication operations resulted in increased functional specialization in network modules \citep{Calabretta:1998vh,Calabretta:2000tl}.
> These and other computational studies \citep{Ryan:1998gm,Sawai:1999genetic,Sawai:2000comparative,Schmitt:2005bc} corroborate some of the benefits of gene duplication seen in natural systems.

- [ ] @amlalejini depending on target venue, remove / de-emphasize evolutionary computation discussion in intro
- [ ] `multiplayaccess` I think this paragraph will need to be edited/framed based on target venue. E.g., cutting down, adding some extra justification about relevance. I can also take some time to update the references. E.g., depending on audience, "genetic programming" is going to be confusing jargon.
- [ ] `multiplayaccess` Given the dates on all of the references in this paragraph, it definitely reads a little dated. I can also take some time to update the references.

**Comment by mmore500** [comment]
> \subsection{Major Results}
- [ ] @amlalejini If there is a venue where the norm is to provide a major results section (or you want to include it regardless -- which is totally fine by me), the current text in this subsection needs to editing to allow someone who just read the intro to fully follow along. Some of these next two paragraphs are hard to follow with just the context from the prior intro (e.g., what is meant by logic-9, etc.). Could summarize major results in 2-3 sentences, and tack them on to the end of the previous paragraph.

**Comment by mmore500** [comment]
- [ ] @amlalejini suggests

> Using the Avida Digital Evolution Platform \citep{ofria2004avida}, we implemented a series of mutation operators to systematically isolate aspects of gene duplication and tease apart which factors promote evolvability.
> % There are several mechanisms that produce gene duplications in biological systems.
> Here, we use gene duplication mutation operators in Avida that resemble replication slippage \citep{bzymek_instability_2001} (\textit{slip mutations}) and allow for gene duplications or deletions at any scale.
> [Compared evolution of complex traits under different slip mutation operators].
> [Analyzed lineages of evolved digital organisms to identify whether duplication mutations lead to the evolution of new traits].
> % AML (2025): These next two sentences were redundant. Added alternative bullet points above.
> % We introduce five slip mutation operators to tease apart the specific components of a gene duplication.
> % By observing how each of our slip mutation operators affects the evolution of digital organisms in different contexts, we are able to isolate which aspects of gene duplications are most important for promoting evolvability.

**Comment by mmore500** [comment]
> The prominence of gene duplications in biological evolution has inspired their use in artificial evolutionary systems.
> In genetic programming, the optimal program architecture for solving a particular problem is challenging to predict \textit{a priori}.
> Inspired by Ohno's \textit{Evolution by Gene Duplication} \citep{ohno1970evolution}, Koza used gene duplication and deletion operators to co-evolve genetic programs and their structure, finding that gene duplication and deletion increased program evolvability and produced simpler solutions \citep{Koza:1995fr}.
> Calabretta \textit{et al.} evolved modular neural network motor controllers for robots with and without module duplication operators, finding evidence that access to duplication operations resulted in increased functional specialization in network modules \citep{Calabretta:1998vh,Calabretta:2000tl}.
> These and other computational studies \citep{Ryan:1998gm,Sawai:1999genetic,Sawai:2000comparative,Schmitt:2005bc} corroborate some of the benefits of gene duplication seen in natural systems.

- [ ] `multiplayaccess` Given the dates on all of the references in this paragraph, it definitely reads a little dated. I can also take some time to update the references.
